java_gen/pre-written/pom.xml

```diff
@@ -131,20 +131,7 @@
                     </execution>
                 </executions>
             </plugin>
-            <!-- attach javadoc -->
```

Comment:
Would be good to retain this. This gives us the `-javadoc` jar on Maven that IDEs like Eclipse use to present richer inline help to the user.

@abakagamze - what error do you get from the plugin?